src/config/locales/en/error.js

```diff
@@ -497,6 +497,12 @@ export const error = {
           message:
             "For the **from** date, the date should be after this person's date of birth."
         }
+      },
+      foreignNationalSponsorship: {
+        min: {
+          title: 'There is a problem with your entry',
+          message: 'For the **from** date, the date should be after their date of birth.'
+        }
```

Comment:
When you add a prefix, it prefixes _all_ of the error messages associated with that component unfortunately, so adding in a `required` message as well in case the date field is left blank.
```suggestion
        },
        required: {
          title: 'Your response is required',
          message: ''
       }
```